pkg/storaged/storage.scss

```diff
@@ -626,19 +626,20 @@ td.storage-action {
 
 /* Overview side panels */
 
-#storage .storage-sidebar table {
-    table-layout: fixed;
+.sidepanel-row {
+    padding-left: var(--pf-global--spacer--md);
+    padding-right: var(--pf-global--spacer--md);
+    padding-top: var(--pf-global--spacer--sm);
+    padding-bottom: var(--pf-global--spacer--sm);
+    border-bottom: var(--pf-global--BorderWidth--sm) solid var(--pf-global--BorderColor--100);
 }
 
-.sidepanel-row {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    gap: var(--pf-global--spacer--sm);
+.sidepanel-row:last-child {
+    border-bottom: none;
 }
 
-.sidepanel-row-detail {
-    margin-right: 0.5em;
+.sidepanel-row:hover {
+    background-color: var(--color-link-hover-bg);
```

Comment:
This variable will change soon in PR https://github.com/cockpit-project/cockpit/pull/15183 (which removes a _lot_ of Cockpit-specific CSS variables and standardizes the remaining ones with a prefix)

Specifically, to `--ct-color-link-hover-bg`
https://github.com/cockpit-project/cockpit/pull/15183/files#diff-7d0ccb2f6de8d81d3e61bc5c350c8b0461851c05480bee0f5d715d4ee8001909R309

I guess I need to finish up that PR. :wink:

Reply:
Note: This shouldn't be updated until that lands. Else, it should stay the same and be updated in that other PR. (Whichever PR lands second of these two will need to change it, in other words.)

Reply:
Can I use that prefix already here?

Reply:
> We'll have to change the color at some point and should change the padding to shorthand. I still need to review this locally; right now, I just glanced at the diff (mainly focusing on the CSS).

@garrett What does it mean we 'have to change the color'? You mean what table-hover used to do? If that's the case, isn't having links enough?

Reply:
@KKoukiou it means to change the name of the variable; it's changing in PR #15183